.github/workflows/check-diamond-storage.yml

```diff
@@ -0,0 +1,38 @@
+name: Diamond Storage Check
+
+on:
+  pull_request:
+
+jobs:
+  check-diamond-storage:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: 14
+
+      - name: Install dependencies
+        run: npm install
+
+      - name: Setup Foundry
+        uses: foundry-rs/foundry-toolchain@v1
+        with:
+          version: nightly-87bc53fc6c874bd4c92d97ed180b949e3a36d78c
```

Comment:
we have a stable. hash now, https://github.com/ubiquity/ubiquity-dollar/actions/runs/5476516361/workflow#L29, for the foundry version